FinniversKit/Sources/Base Components/ViewModelCoordinator/UnownedViewModelCoordinator.swift

```diff
@@ -0,0 +1,35 @@
+// MARK: - ViewModelCoordinator + Extension
+
+extension ViewModelCoordinator {
+    public func eraseToUnownedCoordinator() -> UnownedViewModelCoordinator<Route> {
+        return .init(coordinator: self)
+    }
+
+    public func eraseToUnownedCoordinator<TargetRoute>(transform: @escaping (TargetRoute) -> Route) -> UnownedViewModelCoordinator<TargetRoute> {
+        return .init(coordinator: self, transform: transform)
+    }
+}
+
+// MARK: - UnownedCoordinator
+
+public final class UnownedViewModelCoordinator<Route>: ViewModelCoordinator {
```

Comment:
When is the unowned version needed? How is the lifetime of the wrapped coordinator ensured to be as long as the unowned coordinator?

Reply:
It depends on the use case. I didn't needed this, but there might be cases later when you want to avoid circular dependencies and this could help someone. 😉